src/test/java/de/retest/recheck/RecheckImplIT.java

```diff
@@ -79,9 +79,12 @@ void execute( final String name, final RootElement toVerify, final RecheckOption
 
 		re.startTest( name );
 		re.check( toVerify, new RootElementAdapter(), "check" );
-		re.capTest();
 
-		re.cap();
+		try {
+			re.capTest();
+		} finally {
+			re.cap();
```

Comment:
Why not in `@AfterEach`?

Reply:
I emulate the lifecycle of Recheck within a test twice so that a error message can be produced.